### PR TITLE
[release-1.11] Align w/ the KnativeServing and KnativeEventing

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

We do it like `KnativeServing` and `KnativeEventing`.

Fix for: https://issues.redhat.com/projects/SRVKE/issues/SRVKE-616

### CODE REVIEW

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -31,11 +31,13 @@ metadata:
             "name": "knative-kafka"
           },
           "spec": {
-            source:
-              enabled: true
-            channel:
-              enabled: true
-              bootstrapServers: REPLACE_WITH_COMMA_SEPARATED_KAFKA_BOOTSTRAP_SERVERS
+            "source": {
+              "enabled": "false"
```
Comment (maschmid) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:35]: Are you sure it should be a string "false", and not a boolean false?

Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:35]: @maschmid JSON fu :-) 

Before I sent the PR I looked here, how we used to do that in the past for the `alm-examples`:
https://github.com/openshift-knative/knative-serving-operator/blob/v0.8.0/deploy/olm-catalog/knative-serving-operator/0.8.0/knative-serving-operator.v0.8.0.clusterserviceversion.yaml#L5

that uses `"true"` syntax too :man_shrugging:

Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:35]: and `"false"` is also twice in there.

So, we have done strings for booleans on this dialect before

**Pull request merged**